components/api/src/main/java/com/hotels/styx/api/CookieHeaderNames.java

```diff
@@ -0,0 +1,37 @@
+/*
+  Copyright (C) 2013-2020 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.api;
+
+final class CookieHeaderNames {
```

Comment:
Should this be called `CookieAttributeNames` instead?

Reply:
This was copied from netty so I was reluctant to rename any of the copied over classes.

Reply:
@mikkokar , what do you think?

Reply:
I would keep the original name.